packages/terra-spacer/src/Spacer.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { withBps } from 'react-bps';
+import 'terra-base/lib/baseStyles';
+import styles from './Spacer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes.
+   */
+  children: PropTypes.node,
```

Comment:
Should we make children required? I'm not sure if this component would make much sense to use without a child element.

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/367486d5156666e74774ec8ba6afcee458336ac9#diff-de1008cdf5bf24f4e07b96189a459cadR14